src/App.scss

```
@@ -1 +1,22 @@
-// style go here
+.article {
+  box-sizing: border-box;
+  padding: 15px;
+  font-family: Arial, Helvetica, sans-serif serif;
+  font-size: 20px;
+  background-color: rgb(241, 235, 235);
+  border-radius: 30px;
+
+  &__title {
+    font-size: 35px;
+  }
+
+  &__date {
+    font-style: italic;
+    color: gray;
+  }
+
+  &__text {
+    line-height: 1.25em;
+    font-family: Arial, Helvetica, sans-serif serif;
+  }
+}
```

Always add a newline at the end of your files
```suggestion
}

```